.github/workflows/README.md

```diff
@@ -57,6 +67,16 @@ This integration test means that you have to wait until the glue workflow has fi
 
 Here that we are **NOT** configuring a S3 event notification configuration for our `prod/staging` space because we plan to submit data to `staging` "manually" after merging a PR into main and triggering the GitHub workflow.
 
+### integration-test-staging-cleanup
+
+This is responsible for cleaning up any data locations that are used by integration
+tests during the staging run. This is used after `sceptre-deploy-staging`, but before
+`integration-test-staging`. Cleans these locations:
+
+* `s3://recover-dev-input-data/staging/`
```

Comment:
I think this meant to say `recover-input-data` and `recover-intermediate-data`?

Reply:
Great catch